test/specs/sObjectsDefinitions.e2e.ts

```diff
@@ -86,6 +90,9 @@ describe('SObjects Definitions', async () => {
   });
 
   step('Refresh SObject Definitions for Custom SObjects', async () => {
```

Comment:
not related to this PR, but looks like we can refactor out the step method in this test as it seems to be doing the same thing.

Reply:
There are three paths to check All, Standard and Custom and given we do not have unit tests how else do we confirm that the command is setup correctly?